contrib/docker-compose/standalone/haproxy/etc/haproxy-internal.cfg

```diff
@@ -0,0 +1,56 @@
+global
+    log stdout format raw local0
+
+    # Allow local admin socket
+    stats socket /usr/local/etc/haproxy/haproxy.admin.sock mode 660 level admin expose-fd listeners
+    stats timeout 30s
+
+defaults
+    log global
+
+    option dontlognull
+    option http-server-close
+    option splice-response
+    option clitcpka
+    option srvtcpka
+    option tcp-smart-accept
+    option tcp-smart-connect
+    option contstats
+    retries 3
+
+    timeout http-request    5s
+    timeout http-keep-alive 5s
+    timeout connect         5s
+    timeout client          60s
+    timeout client-fin      60s
+    timeout tunnel          40m         # timeout to use with WebSocket and CONNECT
+    timeout server          150s
+    timeout tarpit          15s
+    timeout queue           10s
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Peerings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+peers haproxy-peers
+    peer "${HOSTNAME}"      "${HOST_IP}":1395
```

Comment:
These envars are defined in the custom haproxy entrypoint, but it's not used for the internal haproxy? Will these still work?

Reply:
Good spot! I've added the entrypoint script.

Reply:
... but the custom entrypoint script is the one that does the periodic restart. Do we want the internal haproxy to restart?

Reply:
I have now added `RELOAD_INTERVAL` logic